Workflow/rebuild_cache

```diff
@@ -49,3 +56,4 @@ Gdrive_paths.each do |gdrive_path|
 end
 
 Tmp_file.rename(Cache_file)
+File.delete(Paths_file) if File.exist?(Paths_file)
```

Comment:
The point of `pathname` is to do these in a different way (e.g. `Paths_file.exist?`).